src/couch_replicator_scheduler.erl

```diff
@@ -129,28 +142,23 @@ handle_cast(_, State) ->
 
 
 handle_info(reschedule, State) ->
-    ok = reschedule(State#state.max_jobs, State#state.max_churn),
+    ok = reschedule(State#state.max_jobs, State#state.max_churn, State#state.max_history),
     {ok, cancel} = timer:cancel(State#state.timer),
     {ok, Timer} = timer:send_after(State#state.interval, reschedule),
     {noreply, State#state{timer = Timer}};
 
-handle_info({'DOWN', _Ref, process, Pid, normal}, State) ->
-    case job_by_pid(Pid) of
-        {ok, #job{}=Job} ->
-            remove_job_int(Job);
-        _Else ->
-            ok
-    end,
+handle_info({'DOWN', _Ref, process, _Pid, normal}, State) ->
```

Comment:
Check latest `63012-scheduler` branch (rebase perhaps), there were code changes there related to monitoring there.